src/main/java/dat3/voximovies/dto/CinemaResponse.java

```diff
@@ -0,0 +1,36 @@
+package dat3.voximovies.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import dat3.voximovies.entity.Cinema;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CinemaResponse {
+    long id;
+     String description;
+    //list of reviews
+    //list of shows
+    //list of seats
+     String street;
+     String city;
+     String zip;
+     String user; //temp
+    Double rating;
+    //User contactPerson;
+    Integer numberOfRatings;
+    public CinemaResponse(Cinema c){
```

Comment:
Jeg tænker at man i stedet for numberOfRatings kan bruge antallet af reviews i listen. Lars kan også godt lide de der includeAll booleans i responset, men ved ikke om det kan give mening her..